components/Banner/examples/supplemental/MDCBannerView.m

```diff
@@ -14,24 +14,39 @@
 
 #import "MDCBannerView.h"
 
-#import "MDCBannerViewLayout.h"
 #import "MaterialButtons.h"
+#import "MaterialTypography.h"
+
+typedef NS_ENUM(NSInteger, MDCBannerViewLayoutStyle) {
+  MDCBannerViewLayoutSingleLineStyle = 0,              // All elements lays on the same line
```

Comment:
copyedit of comment: All elements on the same line

Reply:
Addressed in 070341d4e3cb361c899765e68b6be6ac8e7266aa.